.github/workflows/django.yml

```diff
@@ -13,6 +13,18 @@ jobs:
     strategy:
       max-parallel: 4
 
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: github_actions
+        ports:
+          - "5432:5432"
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
```

Comment:
Nice! I didn't know this existed! https://ryaneschinger.com/blog/using-docker-native-health-checks/